Add io.github.taboulet.BiggerTask

Taboulet opened the pull request:
<!-- ⚠️⚠️  Submission pull request MUST be made against the `new-pr` **base branch** ⚠️⚠️  -->

<!-- 💡 Please go to the preview tab to view the markdown below 💡 -->

### Please confirm your submission meets all the criteria

<!-- 💡 Please replace each `[ ]` with `[X]` when the step is complete 💡 -->

<!-- 💡 Please tick and write 'N/A' with a reason if a checklist item below is not applicable 💡 -->

- [X] Please describe the application briefly. <This app is basically a Linux Clone of the Windows app "Tinytask", it's written in c++ and is a simple executable allowing you to : Record events (Mouse movement/clicks, keyboard inputs) Play them Save them for later Load them to reuse them>
- [X] Please attach a video showcasing the application on Linux using the Flatpak. <https://github.com/user-attachments/assets/3803bd89-4ce5-4280-81f1-0034a862ed76>
- [X] The Flatpak ID follows all the rules listed in the [Application ID requirements][appid].
- [X] I have read and followed all the [Submission requirements][reqs] and the [Submission guide][reqs2] and I agree to them.
- [X] I am an _(please keep whichever is applicable and remove the rest)_ author/developer/upstream contributor to the project.
      If not, I contacted upstream developers about this submission. **Link:**

<!-- 💡 Please mention below the GitHub usernames of any additional maintainers needed (if any) 💡 -->

<!-- ⚠️⚠️  Please DO NOT modify anything below this line ⚠️⚠️  -->

[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[reqs2]: https://docs.flathub.org/docs/for-app-authors/submission

Taboulet commented:
bot, build

Taboulet commented:
bot, build

Taboulet commented:
bot, build

Taboulet commented:
bot, build

Taboulet commented:
bot, build

Taboulet commented:
It works it just has metainfo issues, i'll fix it tmrw

bbhtt commented on io.github.taboulet.BiggerTask.yaml:
```diff
@@ -0,0 +1,33 @@
+      - type: file
+        path: io.github.taboulet.BiggerTask.svg
+      - type: file
+        path: io.github.taboulet.BiggerTask.metainfo.xml
```
these should be in your upstream repo

bbhtt commented on io.github.taboulet.BiggerTask.yaml:
```diff
@@ -0,0 +1,33 @@
+  - name: biggertask
+    buildsystem: simple
+    build-commands:
+      - qmake BiggerTask.pro
```
qmake buildsystem exists.

Taboulet replied:
Fixed

hfiguiere replied:
not fixed.

Taboulet commented:
But so i have to put the .desktop .metainfo.xml and .svg inside [My repo](https://github.com/Taboulet/BiggerTask) and so i just keep the yaml inside the branch ?

hfiguiere commented on io.github.taboulet.BiggerTask.yaml:
```diff
@@ -0,0 +1,33 @@
+sdk: org.kde.Sdk
+command: BiggerTask
+finish-args:
+  - --share=network
```
to clone what?

Taboulet commented:
should be fixed

Taboulet commented:
Is it good or is there still some issues ?

Taboulet commented:
bot, ping mods

Taboulet commented:
I removed nproc, i made it build : qmake, the executable is installed via the the build system, everything except the yaml is in the upstream repo, added the screenshot to the repo for better link,  fixed version of the runtime and remove the C++ mention bcz it's useless

Taboulet commented:
my bad

Taboulet commented:
i added the license to the repo and made a release to be sure its packed with the project
